androidApp/src/main/java/dev/johnoreilly/confetti/di/KoinTestApp.kt

```diff
@@ -0,0 +1,25 @@
+package dev.johnoreilly.confetti.di
+
+import android.app.Application
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.loadKoinModules
+import org.koin.core.context.unloadKoinModules
+import org.koin.core.module.Module
+
+class KoinTestApp : Application() {
```

Comment:
If I get rid of koin test stuff, I have to remove this to

Reply:
Let's leave.